docs/integration/ingestion_methods/syslog/sekoiaio_forwarder.md

```diff
@@ -324,6 +324,95 @@ tls_cert_name: server.crt
 tls_ca_name: server.crt
 ```
 
+## Monitor your concentrator
+
+The forwarder is a critical component in the architecture between your information system and the Sekoia platform. A prolonged service interruption could lead to data loss, potentially causing missed detection of an attack within your environment.
+
+In this context, please find below the instructions to enable monitoring of your forwarder. 
+This will allow health status information of the component to be transmitted to Sekoia, enabling you to set up alerts based on the values of the transmitted metrics.
+
+### Create the forwarder logs intake 
+
+The first step is to create the intake on the Sekoia platform and save the associated intake key
+
+For detailed information on this process, please refer to the following [documentation](/integration/categories/applicative/sekoiaio_forwarder_logs/)
+
+### Configuration of the intake.yml file
+
+To activate the monitoring mode, simply defined a new intake and add `stats: True` in the `intakes.yaml` file with the associated `intake_key` coming from the right format:
+
+```yaml
+---
+intakes:
+- name: Techno1
+  protocol: tcp
+  port: 20516
+  intake_key: INTAKE_KEY_FOR_TECHNO_1
+- name: Monitoring
+  stats: True
+  intake_key: INTAKE_KEY_FOR_FORWARDER_LOGS
+```
+
+!!! Note
+    This is quite a special intake because logs are auto-generated by the forwarder. As a consequency, you don't need to define an input port neither the protocol.
```

Comment:
```suggestion
    This intake is special because logs are auto-generated by the forwarder. As a consequency, you don't need to define an input port neither the protocol.
```